tests/integration/test_hitl_loop_resume_simple.py

```
@@ -0,0 +1,302 @@
+    agent_step = Step(name="agent", agent=StubAgent(["agent_output"]))
+
+    hitl_step = HumanInTheLoopStep(name="hitl", message_for_user="Respond please")
+
+    # Create pipeline with both steps
+    body_pipeline = agent_step >> hitl_step
+
+    # Create loop that exits after one iteration
+    loop = LoopStep(
+        name="test_loop",
+        loop_body_pipeline=body_pipeline,
+        exit_condition_callable=lambda output, context: output == "user_input",
```

Ruff (0.13.3)

Unused lambda argument: `context`

(ARG005)